tutorials/how_to/elmo.md

```diff
@@ -0,0 +1,161 @@
+
+Using pre-trained ELMo representations
+--------------------------------------
+
+Pre-trained contextual representations from large scale bidirectional
+language models provide large improvements for nearly all supervised
+NLP tasks.
+
+This document describes how to add ELMo representations to your model using `allennlp`.
+We also have a tensorflow implementation [here](https://github.com/allenai/bilm-tf).
+
+Reference: ["Deep contextualized word representations"](https://openreview.net/forum?id=S1p31z-Ab)
+
+
+## Installing
+
+After installing `allennlp`, download the pre-trained options and weight files.
```

Comment:
I think you can delete this section (see future comments), but if not it'd be great to have a one sentence description of what each of the files represent.